docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubeedgeadm kubeEdge installer scope (Issue 324)
+status: Pending
+authors:
+    - "@samy2019"
+approvers:
+  - "@m1093782566"
+  - "@rohitsardesai83"
+  - "@sids-b"
+creation-date: 2019-04-11
+last-updated: 2019-04-12
+---
+
+## Motivation
+Many users shared their feedback that kubeEdge installation is too complicated and it may prevent people from trying kubeEdge.
+There should a simplified way to have **Getting Started with KubeEdge**, so that user can concentrate more on using it instantly, rather than
+getting entangled in the installation steps.
+
```

Comment:
Its good to add goals and non-goals . See https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd.md#goals for reference.

Reply:
Added Inscope and out of scope section, please check.